sdk/python/tests/test_util.py

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
```

Comment:
should we not move these tests into our standard compiler test suite?

Reply:
please move the two functions `test_workflow_without_decorator` and `test_nested_workflow` into the actual compiler unit test and have the `test_util.py` call them and capture the output YAML into a file, like `dsl-compile` does